assets/javascripts/discourse/templates/components/topic-footer-buttons.hbs

```diff
@@ -0,0 +1,62 @@
+<div class='topic-footer-main-buttons'>
```

Comment:
What is this?
This is not related to the feature described right?

Reply:
yes it is, this is the reply button at the bottom of the topic (the green one in the video).

By default is not shown when you don't have permissions to post. We need to show it so users know what do to post. 

I'll add the same tooltip we use in the post reply button.